### PR TITLE
pandas_issue6

### PR DESCRIPTION
**Pull request by Harveyt47**
Hi @jstac , update ``ticker_data`` to 2019 data.

### CODE REVIEW

**Comment by jstac** [comment]
Thanks @Harveyt47.

The text mentions that it's 2013 data.  Did you change that as well in another PR?

**Comment by Harveyt47** [comment]
Thanks @jstac , I didn't, I'll add that edit to this PR.

**Comment by shlff** [comment]
Hi @jstac , as I showed to you yesterday afternoon, given your comments, I fixed #889 by 3 points:

1. Read 2019 data live,
2. Change the year from `2013` to `2019`,
3. Modify the solution code by using `np.diff()`.

You can see this updated exercise and its solution in Jupyter-notebook version from my [notes.](https://github.com/shlff/economics_with_python/blob/master/live_data.ipynb). Would you mind confirming these changes?

**Comment by Harveyt47** [comment]
Thanks @shlff , looks good to me.

**Comment by sayaikegawa** [comment]
thanks @shlff , looks good.

File: source/rst/pandas.rst [hunk 1]
```diff
@@ -474,26 +501,11 @@ Exercise 1
...
-                   'PTR': 'PetroChina'}
-
-    price_change = pd.Series()
-
+    price_change = pd.Series(dtype=object)
     for tick in ticker_list:
-        change = 100 * (ticker.loc[ticker.index[-1], tick] - ticker.loc[ticker.index[0], tick]) / ticker.loc[ticker.index[0], tick]
+        a = ticker.loc[ticker.index[0], tick]
+        b = ticker.loc[ticker.index[-1], tick]
+        change = 100 * float(np.diff((a, b))/a)
```
[the remark this answers is not in the document]
Comment (shlff) [reply source/rst/pandas.rst]: Thanks for pointing it out, @jstac . I will have a look at it first, and figure out what to do next.

File: source/rst/pandas.rst [hunk 2]
```diff
@@ -456,13 +464,32 @@ Here's the first line of the program
...
 
-    ticker = pd.read_csv('https://raw.githubusercontent.com/QuantEcon/lecture-source-py/master/source/_static/lecture_specific/pandas/data/ticker_data.csv')
+    def read_data(ticker_list,  
+         start=dt.datetime(2019, 1, 2),
+         end=dt.datetime(2019, 12, 31)): 
+        """
+        This function reads in closing price data from Yahoo 
+        for each tick in the ticker_list.
+        """
+        pd.set_option('precision', 2)
```
[the remark this answers is not in the document]
Comment (shlff) [reply source/rst/pandas.rst]: Hi @jstac , do you think it is fine to simply get rid of the line ``pd.set_option('precision', 2)``?

File: source/rst/pandas.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (Harveyt47) [reply source/rst/pandas.rst]: Thanks @jstac , that speeds things right up. I've modified the solution to just a few lines now.

**Comment by sayaikegawa** [comment]
Thanks @Harveyt47 , that looks good for me.

**Comment by shlff** [comment]
Thanks, @Harveyt47 and @sayaikegawa . Good evening, @jstac , I did 3 things to rescue this PR: 

1. got rid of all other commits in this PR, 
2. brought the branch ``pd6`` in this PR up to date with the most updated master branch, and 
3. modified the Ex1 and its solution according to all our discussions. Now, this PR is ready to be merged.

**Comment by jstac** [comment]
Thanks all.  Code looks good now.  Thanks @shlff for cleaning up the PR at the end.

**Pull request merged**